crates/sla/src/lib.rs

```diff
@@ -336,8 +366,22 @@ impl<T: Config> Module<T> {
     }
 
     /// Explicitly set the vault's SLA score, used in tests.
-    pub fn set_vault_sla(vault_id: T::AccountId, sla: SignedFixedPoint<T>) {
-        <VaultSla<T>>::insert(vault_id.clone(), sla);
+    pub fn set_vault_sla(vault_id: &T::AccountId, sla: SignedFixedPoint<T>) {
+        <VaultSla<T>>::insert(vault_id, sla);
+    }
+
+    /// initializes the relayer's stake. Not that this module assumes that once set, the stake
+    /// remains unchanged forever
```

Comment:
Why? Can we not adjust on registration?

Reply:
As far as I can tell, the staked relayer currently does not have the option to change stakes after registration, so I didn't bother to implement extra logic for dealing with stake changes